src/package_manifest.rs

```diff
@@ -129,8 +129,7 @@ impl PackageManifest {
 
         to_visit_package_manifests.push_back(self);
 
-        while !to_visit_package_manifests.is_empty() {
-            let current_manifest = to_visit_package_manifests.pop_front().unwrap();
```

Comment:
Curious, why is this diff marked as `feat`?

Reply:
I think I was on autopilot, thank you for catching that